### ISSUE TITLE
When training with the coffee_martini_withcam13 dataset, len(videos) is not equal to poses_arr.shape[0]?

### ISSUE THREAD

**Issue by zdrac** [seoha-kim/Sync-NeRF#5]
Hello! Thanks for your wonderful work!
When I was training with the coffee_martini_withcam13 dataset you provided, I got an error saying that len(videos) is not equal to poses_arr.shape[0], I wonder if it's because the poses_bounds.npy file in the dataset you provided doesn't contain cam13 information. If so, could you please upload the corresponding.npy file? I would appreciate it very much! 
I also want to ask, except coffee_martini_withcam13, do you modify the video to be out of sync for other plenoptic datasets, or are they exactly the same as dynerf datasets?
Thank you again!

**Comment by seoha-kim** [comment]
Hello.
I appreciate your interest in my research.

I have already uploaded 18 video poses for the coffee_martini_withcam13 scene.
(The coffee_martini scene has 17 poses uploaded, excluding outlier videos as usual.)

If you encountered an error while trying to train vanilla mixvoxel,
please comment out lines 323 to 326 in dataLoader/llff_video.py.
Because this code lines remove the 13th camera of the martini scene.

The unsynchronization method has been applied equally to all scenes, including 13th camera of the coffee_martini scene.

**Comment by zdrac** [comment]
Hello! Thanks for your reply! I have been learning about your project
recently. I'm a little confused about some questions. Can you help me out?
I would appreciate it very much!
I read in your paper that for the blender dataset, “To measure the accuracy
of the time offsets, the mean absolute error (MAE) between the found
offsets and the ground truth offsets is measured in seconds."
But when I read the code of Sync-K-Planes, I didn't find any code about the
ground truth offsets and MAE. Can you tell me in which line and in which
file you put the code for the ground truth offsets and MAE? How do I get
the ground truth offsets from code to check how accurate the training is?
Since I am new to this field, I am not very experienced in many questions.
I would appreciate it very much if you could reply!

**Comment by seoha-kim** [comment]
The GT offsets for cam1 to cam13 in the Unsynchronized Blender Dataset are as follows:

-10, 7, 1, -5, -4, 3, 0, -1, -3, 5, 13, -8, 2

Please calculate the MAE between these GT offsets and the trained cam offsets.

**Comment by zdrac** [comment]
> Hello. I appreciate your interest in my research.
> 
> I have already uploaded 18 video poses for the coffee_martini_withcam13 scene. (The coffee_martini scene has 17 poses uploaded, excluding outlier videos as usual.)
> 
> If you encountered an error while trying to train vanilla mixvoxel, please comment out lines 323 to 326 in dataLoader/llff_video.py. Because this code lines remove the 13th camera of the martini scene.
> 
> The unsynchronization method has been applied equally to all scenes, including 13th camera of the coffee_martini scene.

Hi😊I'm sorry. I tried again later, and just downloaded poses_bounds.npy from your coffee_ martini_ withcam13 dataset. But when I print (poses arr. shape [0]), its result is 17 instead of 18???

**Comment by seoha-kim** [comment]
Sorry for mistaking it. 😅
I uploaded a new pose array.
